miscellaneous/newdeploy-custommetrics/README.md

```diff
@@ -0,0 +1,104 @@
+# Adding custom metrics to newdeploy functions
+
+In this example, we are going to add custom metrics to the hpa defined in the `function-consumer.yaml` spec file. For that, we will setup a strimzi kafka exporter which is going to provide the metrics that we feed to the newdeploy hpa. Then we will use a pod monitor to scrape the metrics from the pods and finally a prometheus adapter which will expose the metrics to our hpa.
```

Comment:
hpa -> HPA

Please move each statement on new line.

Eg.

This is statement 1.
This is statement 2.

Use markdownlint for linting the file.